src/java.base/share/classes/javax/security/auth/Subject.java

```diff
@@ -90,6 +91,40 @@
  * {@code Principal} implementations associated with Subjects
  * must implement {@code Serializable}.
  *
+ * <h2>Deprecated Methods and Replacements</h2>
+ *
+ * <p> The following methods in this class for user-based authorization
+ * that are dependent on Security Manager APIs are deprecated for removal:
+ * <ul>
+ *     <li>{@link #getSubject(AccessControlContext)}
+ *     <li>{@link #doAs(Subject, PrivilegedAction)}
+ *     <li>{@link #doAs(Subject, PrivilegedExceptionAction)}
+ *     <li>{@link #doAsPrivileged(Subject, PrivilegedAction, AccessControlContext)}
+ *     <li>{@link #doAsPrivileged(Subject, PrivilegedExceptionAction, AccessControlContext)}
+ * </ul>
+ * Methods {@link #current()} and {@link #callAs(Subject, Callable)}
+ * are replacements for these methods, where {@code current}
+ * is mostly equivalent to {@code getSubject(AccessController.getContext())}
+ * and {@code callAs} is similar to {@code doAs} except that the
+ * input type and exceptions thrown are slightly different.
+ *
+ * <p><b><a id="sm-not-allowed">These methods behave differently depending on
+ * whether a security manager is allowed or disallowed</a></b>:
```

Comment:
"is allowed or disallowed" but the detail presents them in the reverse order. I think it would be easier to read if the allowed case went first, this goes for all the methods. I understand that disallow is the default but I think a bit easier to present in that order.